ISSUE_TEMPLATE update (#151)
department-of-veterans-affairs/notification-utils: .github/ISSUE_TEMPLATE/story-template.md

We do not need what to write showing up in tickets though, so please do not uncomment those descriptions.
-->

## User Story - Business Need

- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).

<!--_**Required.** Please note you can have more than one story, if applicable. If this is technical work to achieve a larger initiative, the user story should represent that piece of the initiative so it’s clear what problem we are solving._-->

### User Story(ies)

**As a**    <!-- describe the affected user --><br>
**I want**  <!-- describe the need          --><br>
**So that** <!-- describe the outcome]      --> 

### Additional Info and Resources
<!--Always attempt to include additional information.  This could include screenshots, log snippets, links to applicable code files, and/or articles/websites that have relevant info on the issue. Leave blank if n/a.-->

## Engineering Checklist

- [ ] 

## Acceptance Criteria
<!--_**Required**_
- e.g. when I send an email using an identifier, we retrieve the user’s email address in VA Profile and successfully deliver the email to that recipient-->

<!--
## QA Considerations

> _For QA to populate. Leave blank if QA is not applicable on this ticket._

Reviewer: ```suggestion
_Populate with suggested QA actions if applicable. Leave blank if QA is not applicable on this ticket._
```

Could we replace this comment on both templates please?
